src/openforms/registrations/contrib/zgw_apis/options.py

```diff
@@ -573,11 +595,14 @@ def _validate_medewerker_roltype(
         )
 
     if not roltypen:
-        raise serializers.ValidationError(
-            {
-                "medewerker_roltype": _(
-                    "Could not find a roltype with this description related to the zaaktype."
-                )
-            },
-            code="invalid",
-        )
+        errors = {}
+        if medewerker_description:
+            errors["medewerker_roltype"] = _(
+                "Could not find a roltype with this description related to the zaaktype."
+            )
```

Comment:
you can define the error message in a variable and then you don't have to keep the string literals in sync.